src/awkward/_util.py

```diff
@@ -2,18 +2,16 @@
 from __future__ import annotations
 
 import os
+import struct
 import sys
 from collections.abc import Collection
 
 import packaging.version
 
-from awkward._nplikes.numpylike import NumpyMetadata
 from awkward._typing import TypeVar
 
-np = NumpyMetadata.instance()
-
 win = os.name == "nt"
-bits32 = np.iinfo(np.intp).bits == 32
+bits32 = struct.calcsize("P") * 8 == 32
```

Comment:
This is the only non-sentinel change

Reply:
Either way, you'd asking the OS whether `sizeof(size_t)` is equal to 4. `np.iinfo` goes through NumPy's code and `struct.calcsize` goes through Python's, but this is determined by whatever compiled NumPy and Python. (And if they weren't both 32-bit or both 64-bit, lots of things would fail.)

But okay.

Reply:
Yep - the motivation for this is to remove the nplike import in favour of a built-in, so that we can decouple modules (we don't need to, but it's long term preferable)

Reply:
Yep - the motivation for this is to remove the nplike import in favour of a built-in, so that we can decouple modules (we don't need to, but it's long term preferable)!

Reply:
Yep - the motivation for this is to remove the nplike import in favour of a built-in, so that we can decouple modules (we don't need to, but it's long term preferable)!